# HG changeset patch
# User Max Brister <max@2bass.com>
# Date 1340663188 18000
#      Mon Jun 25 17:26:28 2012 -0500
# Node ID 457eb974310bfbaea14aea128d55d4913015346d
# Parent  b23a98ca0e436d89be9b7ad79be19a38eafdcaf6
src/TEMPLATE-INST/Array-jit.cc: Check for LLVM presence

diff --git a/src/TEMPLATE-INST/Array-jit.cc b/src/TEMPLATE-INST/Array-jit.cc
--- a/src/TEMPLATE-INST/Array-jit.cc
+++ b/src/TEMPLATE-INST/Array-jit.cc
@@ -19,16 +19,20 @@ along with Octave; see the file COPYING.
 <http://www.gnu.org/licenses/>.
 
 */
 
 #ifdef HAVE_CONFIG_H
 #include <config.h>
 #endif
 
+#ifdef HAVE_LLVM
+
 #include "Array.h"
 #include "Array.cc"
 
 #include "pt-jit.h"
 
 NO_INSTANTIATE_ARRAY_SORT (jit_function::overload);
 
 INSTANTIATE_ARRAY (jit_function::overload, OCTINTERP_API);
+
+#endif
